build_config

```diff
@@ -41,11 +38,10 @@ cordova-plugin=cordova-plugin-telerik-imagepicker@2.3.3
 cordova-plugin=cordova-plugin-photo-library
```

Comment:
Media and permission  is used for sunbird use cases

Reply:
There is duplicate plugins , hence deleted one